### PR TITLE
feat: ship batch chains to prod

### PR DESCRIPTION
**Pull request by wanglonghong**
## Type of change

- [ ] Docs change / dependency upgrade
- [x] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: ops/mainnet/prod/backend/config.tf [hunk 1]
```diff
@@ -87,6 +105,12 @@ locals {
       "6450786"    = { confirmations = 50 }
       "6778479"    = { confirmations = 100 }
       "1818848877" = { confirmations = 10 }
+      "2053862243" = { confirmations = 10 }
+      "1887071085" = { confirmations = 10 }
+      "1650553709" = { confirmations = 10 }
+      "1635148152" = { confirmations = 10 }
+      "1835365481" = { confirmations = 10 }
+      "1835101812" = { confirmations = 10 }
```
Comment (LayneHaber) [inline ops/mainnet/prod/backend/config.tf:113]:
```suggestion
      "1835101812" = { confirmations = 1 }
```

op fork needs 1

File: ops/mainnet/prod/backend/config.tf [hunk 2]
```diff
@@ -87,6 +105,12 @@ locals {
       "6450786"    = { confirmations = 50 }
       "6778479"    = { confirmations = 100 }
       "1818848877" = { confirmations = 10 }
+      "2053862243" = { confirmations = 10 }
+      "1887071085" = { confirmations = 10 }
+      "1650553709" = { confirmations = 10 }
```
Comment (LayneHaber) [inline ops/mainnet/prod/backend/config.tf:110]:
```suggestion
      "1650553709" = { confirmations = 1 }
```

op fork, needs 1

**Review by LayneHaber — APPROVED** [review]
some carto nits otherwise lgtm

File: ops/mainnet/prod/backend/config.tf (hunk 2 above)
Comment (wanglonghong) [reply ops/mainnet/prod/backend/config.tf]: gotcha

**Review by preethamr — APPROVED** [review]

**Pull request merged**